README.rst

```diff
@@ -122,6 +122,40 @@ You can configure linting severity, max errors, ... as following:
 
     $ vint --color --style ~/.vimrc
 
+And you can see all available options by `--help` as following:
+
+::
+    $ vint --help
+    usage: vint [-h] [-v] [-V] [-e] [-w] [-s] [-m MAX_VIOLATIONS] [-c]
+                [--no-color] [-j] [-t] [--enable-neovim] [-f FORMAT]
+                [--stdin-alt-path STDIN_ALT_PATH]
+                [files [files ...]]
+    
+    Lint Vim script
+    
+    positional arguments:
+      files                 file or directory path to lint
+    
+    optional arguments:
+      -h, --help            show this help message and exit
+      -v, --version         show program's version number and exit
+      -V, --verbose         output verbose message
+      -e, --error           report only errors
+      -w, --warning         report errors and warnings
+      -s, --style-problem   report errors, warnings and style problems
+      -m MAX_VIOLATIONS, --max-violations MAX_VIOLATIONS
+                            limit max violations count
+      -c, --color           colorize output when possible
+      --no-color            do not colorize output
+      -j, --json            output json style
+      -t, --stat            output statistic info
+      --enable-neovim       enable Neovim syntax
+      -f FORMAT, --format FORMAT
+                            set output format
+      --stdin-alt-path STDIN_ALT_PATH
```

Comment:
I do not find that name good.
What about `--stdin-display-name`?  (used by flake8)

Reply:
I feel `--stdin-display-name` is a good name. Fixed at 7b718a5.